tests/functional/cylc-kill/00-multi-hosts-compat/flow.cylc

```diff
@@ -1,23 +1,29 @@
 #!Jinja2
+
 [scheduler]
     UTC mode = True
     [[events]]
         expected task failures = local-1.1, local-2.1, local-3.1, remote-1.1, remote-2.1
+
 [scheduling]
     [[graph]]
-        R1="""
-KILLABLE:start-all => killer
-"""
+        R1 = """
+            # wait for the remote tasks to start before triggering the
+            # local ones in order to factor out remote-init time
+            remote-1:start & remote-2:start => local-1 & local-2 & local-3
```

Comment:
The preparing stage i.e:

* job file writing
* remote-init
* remote file installation

Is taking >60s on my test platform which results on the local jobs finishing before the remote ones have even started.

Reply:
Ouch